src/data/faq_de.json

```
@@ -35,11 +35,81 @@
                         "indexed": true,
                         "accordion": [
                             {
-                                "title": "Hier werden bald generelle Themen zum Ende der Entwicklung der Corona-Warn-App publiziert.",
+                                "title": "Was ist der Hintergrund für das Ende der Entwicklung der CWA?",
                                 "anchor": "ramp_down",
                                 "active": false,
                                 "textblock": [
-                                    ""
+                                    "Angesichts einer gewachsenen Immunität der Bevölkerung, leicht übertragbarer Virusvarianten und der Rückkehr zu einem öffentlichen Leben (weitgehend) ohne Corona-Maßnahmen entfällt momentan der Bedarf an einer App zur Kontaktnachverfolgung. Ab dem 1. Juni 2023 wird die Corona-Warn-App in einen Ruhe-Modus versetzt. Es wird bis einschließlich 30. April 2023 möglich sein, andere Nutzerinnen und Nutzer durch die Corona-Warn-App hinsichtlich eines erhöhten Infektionsrisikos zu warnen.",

```

Das ist irgendwie verwirrend für außenstehende und sollte nochmal angeschaut werden. Fällt mir leider erst jetzt auf, sorry!

Hier wird gesagt, dass man noch bis zum 30.04.2023 in der App warnen kann und die App zum 01.06.2023 in einen Ruhe-Modus versetzt wird. Jetzt stellt sich mir, als Nutzer:in, aber natürlich die Frage, was denn in der App zwischen dem 01.05. und dem 31.05. passiert? Kann ich in dieser Zeit einfach ganz normal alle Funktionen verwenden, nur halt andere nicht warnen? Oder werden mir schon mit dem 01.05. Funktionen "weggenommen" (die Antwort auf diese Frage wird dann weiter unten gegeben, stellt sich aber hier dennoch)?

Meiner Meinung nach beginnt der Ruhe-Modus, nach meinem Stand, ab dem 01.05. und die Entwicklung der CWA endet zum 01.06. Könnt ihr euch das bitte nochmal anschauen, cc @GisoSchroederSAP!

---

Am 1.5. werden weitgehend alle aktiven Funktionen "abgeschaltet". Ab dann wird der Betrieb nur noch bis zum 31.5. aufrecht erhalten, aktive Nutzung (bis auf die Zertifikatsverwaltung) ist dan nicht mehr möglich. Dann ist es schon ziemlich ruhig.
Im Zeitraum zwischen 1. und 31.5. wird nur noch bei dringlichen Sicherheits-kritischen Bugs ein Hotfix nachgeschoben, falls sie bis Mitte Mai gemeldet wurden. Ab 1.6. werden dann alle Verbindungen deaktiviert und die restlichen Entwicklungsteams aufgelöst. Dann ist es noch ruhiger als ruhig, nämlich ganz still.
Die Texte haben seit dem erstem Entwurf in den letzten 6 Wochen mehr als 10 Review-Runden bei Anwälten, in den Behörden und im Ministerium durchlaufen. Etwa nach der dritten oder vierten Runde wurde durch den Auftraggeber zuerst der "Sleep-Modus" eingeführt und drei Runden später durch "Ruhe-Modus" ersetzt. 
Wir haben schriftlich mehrfach darauf hingewiesen, dass beide Begriffe technisch und organisatorisch nicht abgestützt sind.

---

Okay, dann werden wir an den Begriffen nichts mehr ändern, trotzdem sollten wir irgendwie an dieser Stelle schon klarstellen, dass bereits ab dem 01.05. die meisten Funktionen nicht mehr in der App verfügbar sein werden.

Was hältst du von folgendem Vorschlag:
```suggestion
                                    "Angesichts einer gewachsenen Immunität der Bevölkerung, leicht übertragbarer Virusvarianten und der Rückkehr zu einem öffentlichen Leben (weitgehend) ohne Corona-Maßnahmen entfällt momentan der Bedarf an einer App zur Kontaktnachverfolgung. Ab dem 1. Juni 2023 wird die Corona-Warn-App in einen Ruhe-Modus versetzt. Es wird bis einschließlich 30. April 2023 möglich sein, andere Nutzerinnen und Nutzer durch die Corona-Warn-App hinsichtlich eines erhöhten Infektionsrisikos zu warnen. Ab dem 1. Mai 2023 wird der Funktionsumfang der Corona-Warn-App deutlich reduziert.",
```

Sollte das alles aber weitere Abstimmung mit BMG/RKI, etc. bedürfen, lass es mich einfach wissen, dann veröffentlichen wir die Texte erstmal so und bringe das Problem danach intern nochmal auf, bis Mai ist ja doch noch etwas hin.

---

Das wird so nicht klappen. Der einleitende Text wurde vom Produkt-Owner in schwierigen Diskussionen mit dem BMG "arrangiert". Jetzt auf eine "deutliche Reduktion" zu insistieren, wird nicht mehr akzeptiert, sorry

---

@GisoSchroederSAP Okay, dann machen wir an diesen Texten erstmal keine Änderungen.

Danke Giso!

---

**THIS CONVERSATION IS RESOLVED! PLEASE DO NOT ACT ON IT!**